### DISCUSSION TITLE
pychron crash upon startup

### DISCUSSION THREAD

**Discussion by ANGL-40Ar39Ar** [NMGRL/pychron#2166] [Q&A] [answered]
[pychron.current.log](https://github.com/NMGRL/pychron/files/8491253/pychron.current.log)
[pychron-00001.log](https://github.com/NMGRL/pychron/files/8491254/pychron-00001.log)

**Comment by jirhiker** [comment, marked as answer]
see https://github.com/NMGRL/pychron/wiki/Crash-on-Startup

The solution is to delete the `application_memento` file. 

This file is part of the underlying UI/UX framework used by pychron. It stores the application state, essentially which windows are open on startup.  By deleting this file you are effectively doing a "factory reset".
